src/Uno/Prism.Uno/Services/Dialogs/DialogService.WinUI.cs

```diff
@@ -0,0 +1,140 @@
+﻿using Prism.Common;
+using Prism.Ioc;
+using System;
+using System.ComponentModel;
+using System.Linq;
+using System.Windows;
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Prism.Services.Dialogs
+{
+    public class DialogService : IDialogService
+    {
+        private readonly IContainerExtension _containerExtension;
+
+        public DialogService(IContainerExtension containerExtension)
```

Comment:
There's been a change as well here that will start getting implemented on the other platforms. If you only need access to resolve types like in the DialogService you can now inject the IContainerProvider instead.

Reply:
I've updated this.